modules/layers/src/column-layer/column-geometry.js

```diff
@@ -0,0 +1,133 @@
+import {Geometry, uid} from '@luma.gl/core';
+
+const INDEX_OFFSETS = {
+  x: [2, 0, 1],
+  y: [0, 1, 2],
+  z: [1, 2, 0]
+};
+
+export default class ColumnGeometry extends Geometry {
+  constructor(props = {}) {
+    const {id = uid('column-geometry')} = props;
+    const {indices, attributes} = tesselateTruncatedCone(props);
+    super({
+      ...props,
+      id,
+      indices,
+      attributes: {...attributes, ...props.attributes}
+    });
+  }
+}
+
+// Primitives inspired by TDL http://code.google.com/p/webglsamples/,
+// copyright 2011 Google Inc. new BSD License
+// (http://www.opensource.org/licenses/bsd-license.php).
+/* eslint-disable max-statements, complexity */
+function tesselateTruncatedCone(props) {
```

Comment:
This is good. But we can have a followup PR to simplify the geometry. `ColumnGeometry` is a cylinder of which has the same size plane along vertical axis, so it is not necessary to have a vertical segmentation (`nvertical` is not needed).

Reply:
Sounds good. Noted.